---
layout: page
title: My road in Research
permalink: /research/
---

The abundance of genome data being produced by new sequencing techniques has clearly put the field of bioinformatics in the spotlight. Bioinformatics is an interdisciplinary field that develops and improves on methods for storing, retrieving, organizing and analyzing biological data. A such, a major activity in bioinformatics is to develop software tools to generate useful biological knowledge. In this context, my background in electrical and computer engineering coupled with the extensive experience in working on currently open research issues in Life Sciences, has provided me with a unique perspective in both the algorithmic aspects of approaches in bioinformatics, as well as some insights towards potential future directions.

My earliest work was primarily focused on the design and application of data mining algorithms, applicable both to research questions in bioinformatics such as protein classification ({% cite PsomopoulosDiplarisPisa PolychroniadouPsomopoulosGClass CkekasFpsomSamos %}) and molecular dynamics ({% cite MprouzaFpsomSamos %}) as well as within e-infrastructures ({% cite GkekasPsomopoulosEGEEForum FpsomPCI2009 Psomopoulos2010a %}). These works consolidated the knowledge in the efficient design of novel data analysis and modeling algorithms as well as established a long-standing working relationship with European e-infrastructures, leading to my later selection as an EGI Champion in Bioinformatics in 2013.

Building on this knowledge and experience, the next stage in my research was oriented towards a better understanding of the current situation in Life Sciences, by addressing key questions in Comparative Genomics. One of the first novel approaches in this context was the definition of fuzzy phylogenetic profiles and their application in the detection of genomic idiosyncrasies ({% cite FuzzyBMCBioinformatics %}), i.e. sets of genes found in a specific genome with peculiar phylogenetic properties, such as intra-genome correlations or inter-genome relationships. As a method, it is demonstrated to be extremely efficient, both in terms of computational complexity and high scalability with various uses, including as a validation approach for further studies. A second study focused on the definition of the pangenome, and the development of a computational workflow that defined the salient features of the Chlamydiales pangenome and its evolutionary history ({% cite GenesChlamydiae %}). This work included both the definition and development of a robust computational pipeline for pangenomes, as well as allowed for a deeper understanding of the biological concepts underlying the evolutionary histories of the bacterial order under study. Finally, building on both fuzzy phylogenetic profiles and pangenomes, one of the most recent works is PathTrace, an efficient algorithm for parsimony-based reconstructions of the evolutionary history of individual metabolic pathways (_paper under review_). By deploying a pangenome-driven approach, it is demonstrated that the inferred patterns are largely insensitive to noise, as opposed to gene content reconstruction methods. In addition, there is a strong indication that the resulting reconstructions are closely correlated with the evolutionary distance of the taxa under study, suggesting that a diligent selection of target pangenomes is essential for maintaining cohesiveness of the method and consistency of the inference, serving as an internal control for an arbitrary selection of queries.

Going beyond the development and research on novel tools and algorithms in bioinformatics, a third research direction is towards the integration and automation of the bioinformatics workflows, especially within the reproducible environment provided by the e-infrastructures ({% cite KintsakisBigData2016 %}). The long-standing and active involvement within two major e-infrastructures, i.e. [EGI](www.egi.eu) and [RDA](www.rd-alliance.org), allowed for some significant insights into the future steps for computational sciences, leading to a joint perspective article ({% cite Duarte2015 %}). Additionally, the experience in working with large and publicly available datasets, as well as the role of co-chair in the RDA Data Discovery Paradigms Interest Group, led to an in-depth investigation of the use-cases, requirements and best practices for data repositories ({% cite ElevenTips %}), in collaboration with DANS, Elsevier, the Australian National Data Services and the University of Colorado. At the technical level, another research output in the direction is Hermes, a system for the seamless delivery of containerized bioinformatics workflows in hybrid cloud (HTC) environments ({% cite Hermes2017 %}), which combined both the theoretical aspects of workflow management systems in bioinformatics, and leveraged the functionality provided by containers in a cloud environment. An important aspect of this work is the fact that Hermes fosters the reproducibility of scientific workflows by supporting standardization of the software execution environment, thus leading to consistent scientific workflow results and accelerating scientific output.

## References

{% bibliography --cited %}
